replay2/handle_remapper/BUILD.bazel

```diff
@@ -34,16 +42,18 @@ cc_library(
     ),
     visibility = ["//visibility:public"],
     deps = [
-        "//replay2/core_utils",
+        "//replay2/core_utils:core_utils",
+        "//replay2/vulkan_base:vulkan_base",
+        "handle_remapper_src_hdrs"
     ],
 )
 
 cc_test(
-    name = "handle_remapper_test",
+    name = "handle_remapper_tests",
     srcs = [":handle_remapper_src"],
     deps = [
-        "//replay2/core_utils",
-        "//replay2/vulkan_base",
+        "//replay2/core_utils:core_utils",
```

Comment:
nit: Can just leave at `"//replay2/core_utils"` and `"//replay2/vulkan_base",` since that's the shorthand, same below

Reply:
done